### ISSUE TITLE
Tactical VPN connection to the Azure FixNGone tenant

### ISSUE THREAD

**Issue by davidkelliott** [ministryofjustice/modernisation-platform#1460]
## User Story

NOMIS team are keen to start testing connectivity from Azure through to the Modernisation platform.  The existing routes to do this through the tactical and strategic PaloAltos are taking a long time (six weeks plus and counting).  To unblock this we want to create a VPN from MP transit gateway through to Azure FNG

## Value

Unblocks NOMIS work

## Questions / Assumptions

Once we can get connectivity through PTTP and the PaloAltos we start using that again and remove this VPN.

## Definition of done

- [ ] New VPN to Azure FNG
- [ ] New routes in place on MP side
- [ ] New routes in place on FNG side
- [ ] another team member has reviewed
- [ ] tests are green

## Reference

[How to write good user stories](https://www.gov.uk/service-manual/agile-delivery/writing-user-stories)

**Comment by davidkelliott** [comment]
Branch here for work done so far - https://github.com/ministryofjustice/modernisation-platform/compare/feature/vpn-to-azure?expand=1

**Comment by davidkelliott** [comment]
This is no longer needed as we have connectivity via PTTP and the ALZ
